docs/en/latest/tutorials/cache-api-responses.md

```diff
@@ -0,0 +1,222 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This tutorial will focus primarily on handling caching at the API Gateway level by using Apache APISIX API Gateway and you will learn how to use proxy-caching plugin to improve response efficiency for your Web or Microservices API.
+
+**Here is an overview of what we cover in this walkthrough:**
+
+- ✔️ Caching in API Gateway
+- ✔️ About [Apache APISIX API Gateway](https://apisix.apache.org/docs/apisix/getting-started/)
+- ✔️ Run the demo project [apisix-dotnet-docker](https://github.com/Boburmirzo/apisix-dotnet-docker)
+- ✔️ Configure the [Proxy Cache](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/) plugin
+- ✔️ Validate Proxy Caching
+
+## Improve performance with caching
+
+When you are building an API, you want to keep it simple and fast. Once the concurrent need to read the same data increase, you'll face a few issues 😐 where you might be considering introducing **caching**:
+
+- ❌ There is latency on some API requests which is noticeably affecting the user's experience.
+- ❌ Fetching data from a database takes more time to respond.
+- ❌ Availability of your API is threatened by the API's high throughput.
+- ❌ There are some network failures in getting frequently accessed information from your API.
+
+## Caching in API Gateway
+
+[Caching](https://en.wikipedia.org/wiki/Cache_(computing)) is capable of storing and retrieving network requests and their corresponding responses. Caching happens at different levels in a web application:
+
+- Edge caching or CDN
+- Database caching
+- Server caching (API caching)
+- Browser caching
+
+**Reverse Proxy Caching** is yet another caching mechanism that is usually implemented inside **API Gateway**. It can reduce the number of calls made to your endpoint and also improve the latency of requests to your API by caching a response from the upstream. If the API Gateway cache has a fresh copy of the requested resource, it uses that copy to satisfy the request directly instead of making a request to the endpoint. If the cached data is not found, the request travels to the intended upstream services (backend services).
+
+## Apache APISIX API Gateway Proxy Caching
+
+With the help of Apache APISIX, you can enable API caching with [proxy-cache](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/) plugin🔌to cache your API endpoint's responses and enhance the performance. It can be used together with other Plugins too and currently supports disk-based caching. The data to be cached can be filtered with _response codes, request modes_, or more complex methods using the _no_cache_ and _cache_bypass_ attributes. You can specify cache expiration time or a memory capacity in the plugin configuration as well. Please, refer to other `proxy-cache` plugin's [attributes](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/).
```

Comment:
```suggestion
With the help of Apache APISIX, you can enable API caching with [proxy-cache](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/) plugin to cache your API endpoint's responses and enhance the performance. It can be used together with other Plugins too and currently supports disk-based caching. The data to be cached can be filtered with _response codes, request modes_, or more complex methods using the _no_cache_ and _cache_bypass_ attributes. You can specify cache expiration time or a memory capacity in the plugin configuration as well. Please, refer to other `proxy-cache` plugin's [attributes](https://apisix.apache.org/docs/apisix/plugins/proxy-cache/).
```